docs/swagger/report_api.7.yml

```diff
@@ -32,7 +32,7 @@ paths:
             $ref: "#/definitions/QueryParams"
       responses:
         "200":
-          description: "logins reponse"
+          description: "logins response"
```

Comment:
I'm hoping this isn't a problem.

Note that I'm not sure I like `logins`, but I left it alone.

---

I'm also not sure if it's ok to change a swagger file...

Reply:
Changing the description field is ok. Also this swagger file is obsolete now anyway.